requirements/development.txt

```diff
@@ -437,7 +435,7 @@ marshmallow==3.26.1
     #   -c requirements/base.txt
     #   flask-appbuilder
     #   marshmallow-sqlalchemy
-marshmallow-sqlalchemy==0.28.2
+marshmallow-sqlalchemy==1.4.1
```

Comment:
I think this one makes CI fail with out-of-memory issues, need to figure out which exact version brought the issue